EventFilter/EcalRawToDigi/plugins/alpaka/UnpackPortable.dev.cc

```diff
@@ -0,0 +1,457 @@
+// Check that ALPAKA_HOST_ONLY is not defined during device compilation:
+#ifdef ALPAKA_HOST_ONLY
+#error ALPAKA_HOST_ONLY defined in device compilation
+#endif
```

Comment:
I added this as a cross-check in some test files, to check that the `.dev.cc` files were being compiled correctly.

I wouldn't mind keeping it, but it can be removed.

Reply:
I think the use in
https://github.com/cms-sw/cmssw/blob/d59e80b6862b73d0497919044d19841844f164b3/HeterogeneousCore/AlpakaTest/plugins/alpaka/TestAlgo.dev.cc#L1-L4
is fine (and necessary!) as it is test code. But I'd rather trust that test to be sufficient to ensure the build rules than replicate the check elsewhere.

Reply:
I have removed it.